.gitignore

```diff
@@ -50,3 +50,8 @@ src/arm-none-eabi
 
 # clangd
 .cache/
+
+# node
+/node_modules
+/package-lock.json
+/package.json
```

Comment:
These changes are unrelated. Maybe you could make a separate PR for it?

Reply:
Good point, thank you! Didn't mean to lump these into here. Removed changes to .gitignore in https://github.com/InfiniTimeOrg/InfiniTime/pull/1326/commits/8880acc1f69b34b7e89212549b867d9fc2724fd8 .